### PR TITLE
Add wkhtmltox to travis

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by ghost** [comment]
@max3903 @bwrsandman I removed 3 unit tests in mgmtsystem_probability which fail. I don't agree with that kind of test because it might not even increase the coverage and it does not prove anything.

**Comment by max3903** [comment]
:+1:

File: .travis.yml [hunk 1]
```diff
@@ -30,6 +30,9 @@ install:
   - git clone https://github.com/OCA/maintainer-quality-tools.git ${HOME}/maintainer-quality-tools
   - export PATH=${HOME}/maintainer-quality-tools/travis:${PATH}
   - travis_install_nightly
+  - apt-get install xfonts-75dpi
+  - wget http://sourceforge.net/projects/wkhtmltopdf/files/archive/0.12.2/wkhtmltox-0.12.2_linux-precise-amd64.deb
+  - dpkg -i wkhtmltox-0.12.2_linux-precise-amd64.deb
```
Comment (bwrsandman) [inline .travis.yml]: You can't use `apt-get install` inside of a container travis. You need sudo.
Same for dpkg.

Comment (bwrsandman) [inline .travis.yml]: Can you use one of these?
https://github.com/travis-ci/apt-package-whitelist/blob/master/ubuntu-precise#L6079
https://github.com/travis-ci/apt-package-whitelist/blob/master/ubuntu-precise#L6037

They can be added to the `addons: apt: packages:`

Comment (ghost) [inline .travis.yml]: @bwrsandman I thought the ubuntu package had the version with unpatched qt. Is it still the case?

Comment (bwrsandman) [inline .travis.yml]: I don't know... If that's the case, try downloading the binary locally and using an alias or something to avoid needing to use sudo.

Comment (ghost) [inline .travis.yml]: @bwrsandman I am not sure what you mean or how to do that.

Comment (bwrsandman) [inline .travis.yml]:
```
wget http://wkhtmltopdf.googlecode.com/files/wkhtmltopdf-0.11.0_rc1-static-amd64.tar.bz2
tar xj wkhtmltopdf-0.11.0_rc1-static-amd64.tar.bz2
```

then make the binary available to Odoo

**Comment by ghost** [comment]
@bwrsandman Travis is green. I don't know why runbot is red.

**Comment by pedrobaeza** [comment]
This are some warnings from deprecation type field in view and null constraint, so nothing about this PR.

I merge :+1:

**Pull request merged**